test/loader/test_link_neighbor_loader.py

```diff
@@ -213,14 +212,12 @@ def test_temporal_heterogeneous_link_neighbor_loader():
         edge_label_time=edge_time,
         batch_size=32,
         time_attr='time',
-        neg_sampling_ratio=0.5,
+        neg_sampling_ratio=1.0,
```

Comment:
```suggestion
        neg_sampling_ratio=0.5
```
Lets test the case of `amount<1`.